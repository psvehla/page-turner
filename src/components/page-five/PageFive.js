import React from 'react';
import bike from '../../images/5.jpg';

import './PageFive.css';

function PageFive(props) {
  return (
    <div><img src={bike} alt="bike" onClick={() => props.onClick()} /></div>
  );
}

export default PageFive;
