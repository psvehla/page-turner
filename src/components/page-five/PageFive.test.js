import React from 'react';
import renderer from 'react-test-renderer';
import PageFive from './PageFive';

it('renders correctly', () => {

  const tree = renderer
    .create(<PageFive />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
