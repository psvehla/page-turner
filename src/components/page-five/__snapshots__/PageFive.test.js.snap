// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="bike"
    onClick={[Function]}
    src="5.jpg"
  />
</div>
`;
